README.md

```diff
@@ -1 +1,76 @@
-# angular-what-is-a-filter-readme
\ No newline at end of file
+# What is a Filter?
+
+## Overview
+
+Angular offers us the powerful concept of filters. Let's take a look at what they can offer us.
+
+## Objectives
+
+- Show an Angular filter
+- Use a filter on an ng-repeat
+- Use a filter on an input's expressed value
+
+## Filtering our datasets
+
+Filters are simple functions that we can use to manipulate (or filter) data in Angular.
+
+We've touched on the `ng-repeat` directive for repeating a list of data - wouldn't it be neat if we could filter on that data too? This comes in useful when we a large dataset and would like to quickly search through it to find the relevant data.
```

Comment:
nice presentation of the problem here - this shows how this feature will be useful to our users